docs/README.md

```diff
@@ -8,25 +8,55 @@
 
 # Availability Notify
 
-## Description
-
 `AvailabilityNotfier` is a VTEX Component that shows the availability notify form that is shown when the product isn't available.
-The app will record the request for notification and monitor inventory updates.  When the requested sku is back in stock, the app
-will send an email to the shopper that requested to be notified.
 
-## Usage
+The app records the request for notification and monitors inventory updates.  When the requested sku is back in stock, the app will send an email to the shopper that requested to be notified.
+
+## Configuration
+
+1. [Install](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-installing-an-app) the Availability Notify app in the desired VTEX account by running `vtex install vtex.availability-notify` in your terminal.
 
-To use this app, you need to import it in your store theme peer dependencies on manifest.json.
+2. Open your store’s Store Theme app directory in your code editor.
+
+3. Add the Availability Notify app to your theme’s in your store theme peer dependencies on `manifest.json`.
 ```json
   "peerDependencies": {
     "vtex.availability-notify": "0.x"
   }
 ```
-Then, you can add the `availability-notify` component block to your PDP in your store theme.
+4. Add the `availability-notify` component block to your PDP in your store theme (`store.product`). For example:
+
+```json
+{
+ "store.product": {
+    "children": [
+      "availability-notify"
+    ]
+  },
+
+```
+
+
+>ℹ️ The email template that will be used is `back-in-stock`
```

Comment:
We should specify or link to some other document explaining what is included in this email template and how to change it.